cupy/core/elementwise.pxi

```diff
@@ -365,12 +365,13 @@ cdef list _get_out_args(list out_args, tuple out_types, tuple out_shape,
 
 
 cdef list _get_out_args_with_params(
-        list out_args, tuple out_types, tuple out_shape, tuple out_params):
+        list out_args, tuple out_types, tuple out_shape, tuple out_params, size):
```

Comment:
When "tuple size=None" is used, I get the following error: 

    Traceback (most recent call last):
      File "test_user-kernel.py", line 27, in <module>
        out2 = uesr_kernel_2( in1.data, in2.data, size=n )
      File "cupy/core/elementwise.pxi", line 526, in cupy.core.core.ElementwiseKernel.__call__ (cupy/core/core.cpp:35984)
    TypeError: Expected tuple, got int